### ISSUE TITLE
Clarify "Free Trial!"

### ISSUE THREAD

**Issue by marvinmarnold** [NebraLtd/maker-starter-app#153]
Based on some of the store reviews, users seem to think the app is a paid product. We should clarify what the distinction between the dashboard and app and what's paid.

> We already payed for the machine that doesn't work/perform as desired and arrived 6 months later, and now we have to pay more for an app? Really Nebra, be ashamed!

Possibly merge into: https://github.com/NebraLtd/maker-starter-app/issues/152

Acceptance criteria
- When link is clicked, it opens the embedded Dashboard sign up page
- Title: Advanced Fleet Management
- Subtitle: Remotely control and monitor the health of all your Nebra hotspots.

**Comment by Andrewsopoku** [comment]
@marvinmarnold your input is required here

**Comment by marvinmarnold** [comment]
@Andrewsopoku, I just updated the text in the AC

**Comment by Andrewsopoku** [comment]
@marvinmarnold Seen thanks. But is the dashboard home page not more appropriate than the sign up page? A user may already be signed in or have an account.

**Comment by marvinmarnold** [comment]
Published v1.0.5 to both stores. Waiting on approval.
